### PR TITLE
Full gpu inference pipeline

### PR DESCRIPTION
**Pull request by zmsunnyday**
Fixes # .

### Description
This example implements a 3D medical imaging AI inference pipeline using MONAI and Triton. The goal is to combine Triton python backend BLS and GPU accelerated data processing operators in MONAI to build a full GPU inference pipeline, and show the performance improvement MONAI can bring to inference.
### Status
**WIP**

### Checks

- [ ] Notebook runs automatically `./runner [-p <regex_pattern>]`

### CODE REVIEW

File: full_gpu_inference_pipeline/README.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+### Prepare the model repository file directories
+The model repository of the experiments can be fast set up by: 
+```bash
+git clone https://gitlab-master.nvidia.com/menzhang/triton_monai.git
+cd triton_monai/
+```
+The model repository is in folder triton_models. The file structure of the model repository should be:
+```
+triton_models/
```
Comment (wyli) [inline full_gpu_inference_pipeline/README.md:39]: thanks, could you please 

- put the model folder `triton_models/*` on google drive, and share only a URL here, like this data section https://github.com/Project-MONAI/tutorials/tree/master/self_supervised_pretraining#1data
- update to have a monai copyright section at the beginning of the files? https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md#checking-the-coding-style
- there are some gitlab links which won't work properly...

Comment (zmsunnyday) [reply full_gpu_inference_pipeline/README.md:43]: Hi, thanks for your comments. 
-  Although `triton_model` is the model folder, there are code and config files, which I think would be better put on Github.
- I have updated MONAI copyright section at the beginning of code files.
- The Gitlab links are removed and replaced with the right links.
- Code style has been checked by following https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md#checking-the-coding-style, and passed. 
Please feel free to let me know if there is anything that should be modified.

Comment (wyli) [reply full_gpu_inference_pipeline/README.md:43]: Thank you, it looks good. Could you please replace the URLs of your personal pages (such as https://github.com/zmsunnyday/tutorials/raw/full_gpu_inference_pipeline/full_gpu_inference_pipeline) with the project-monai links?

for the model files, if you could send me a zip or tar file, I can upload it to https://github.com/Project-MONAI/MONAI-extra-test-data/releases/tag/0.8.1 where we agreed to use it for testing/tutorial data storage. Sorry about this inconvenience but if we all include the files in this codebase, it becomes very large and not easily managable quickly.

Comment (zmsunnyday) [reply full_gpu_inference_pipeline/README.md:41]: Hi,
- I have replaced my personal repo links with project-MONAI links in README.md.
- I have uploaded the zip file [here](https://drive.google.com/file/d/1NPlbS7jmpmRE_4ey31SdveKBBKX9CFS9/view?usp=sharing). Please let me know if you have access to the file.
Thanks.

Comment (wyli) [reply full_gpu_inference_pipeline/README.md:41]: thanks, I uploaded the model to https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/triton_models.zip

Comment (zmsunnyday) [reply full_gpu_inference_pipeline/README.md:42]: Updated. Please review, thanks.

**Review by wyli — APPROVED** [review]
Thank you, it looks nice...

**Pull request merged**